test_system_nidmm.py's test_acquisition is broken

```
src\nidmm\system_tests\test_system_nidmm.py:108: NameError
______________________________ test_acquisition _______________________________

device_info = {'name': '4070', 'sn': '00D82BA5'}

    def test_acquisition(device_info):
        with nidmm.Session(device_info['name']) as session:
            session.configure_measurement_digits(nidmm.Function.DC_CURRENT, 1, 5.5)
>           with session.acquisition:
E           AttributeError: 'Session' object has no attribute 'acquisition'

src\nidmm\system_tests\test_system_nidmm.py:122: AttributeError
===================== 4 failed, 7 passed in 4.92 seconds ======================
```

--- a/src/nidmm/system_tests/test_system_nidmm.py
+++ b/src/nidmm/system_tests/test_system_nidmm.py
@@ -116,6 +116,7 @@ def test_ViSession_attribute(device_info):
             print(e)
             pass
 
+
 def test_acquisition(device_info):
     with nidmm.Session(device_info['name']) as session:
         session.configure_measurement_digits(nidmm.Function.DC_CURRENT, 1, 5.5)
@@ -123,6 +124,7 @@ def test_acquisition(device_info):
             print(session.fetch(1000))
         with session.initiate():
             print(session.fetch(1000))
+
 
 '''
 def test_self_test(device_info):

--- a/src/nidmm/system_tests/test_system_nidmm.py
+++ b/src/nidmm/system_tests/test_system_nidmm.py
@@ -119,11 +119,11 @@ def test_ViSession_attribute(device_info):
 def test_acquisition(device_info):
     with nidmm.Session(device_info['name']) as session:
         session.configure_measurement_digits(nidmm.Function.DC_CURRENT, 1, 5.5)
-        with session.acquisition:
+        with session.initiate():
             print(session.fetch(1000))
-        with session.acquisition:
+        with session.initiate():
             print(session.fetch(1000))
-       
+
 '''
 def test_self_test(device_info):
     with nidmm.Session(device_info['name']) as session:

